### ISSUE TITLE
Display [Instrumental] label on songs that are verified instrumental

### ISSUE THREAD

**Issue by shipurjan** [tranxuanthang/lrcget#36]
Right now it only says [Synced] or [Plain]

**Comment by tranxuanthang** [comment]
I think we need to save a file to persist the instrumental status even after rescanning of directories. Two possible options are:

- We save a .lrc file with the content contains only a custom invented tag name like `[ins:true]`, marking the track that has the same name as instrumental
- Or simply save an empty .txt file with the same name

**Comment by lightmaster** [comment]
Could also save a .lrc with `(Instrumental)` in it. That way if you're looking through your collection on whatever media player you use, you won't see it is missing lyrics. There are plenty of songs with titles that wouldn't immediately hint that it's probably instrumental.

**Comment by Dr-Blank** [comment]
as of now there is no metadata tag for marking the track to be instrumental in .lrc format afaik.

Since instrumental tracks have no lyrics, empty lyrics file makes logical sense. but it also needs to be discernable that the lyrics are empty on purpose and not empty because of a bug. hence some sort of metadata tag should be used to indicate that the file is empty on purpose.

i suggest using the existing author tag "au" for this insteadd of creating a new one.

eg metadata would be
```lrc
[ar: Jefferson Airplane]
[al: Surrealistic Pillow]
[au: instrumental]
[length: 2:58]
```

**Comment by tranxuanthang** [comment]
@Dr-Blank It makes sense, I prefer not inventing new standards, and unconventional tag names might potentially break some faulty clients.

I agree with the solution of adding an LRC file that contains `[au: instrumental]`, although I don't think adding other metadata tags (`ar`, `al`, `au`, ...) is necessary.

If someone could make a PR for this, it would be greatly appreciated!

**Comment by tranxuanthang** [comment]
Good news: it is kind of working after commit https://github.com/tranxuanthang/lrcget/commit/754dd0f514ced70318e5982e7474d5fefae0321c, though I suspect some more work needs to be done in order to release this.

**Comment by tranxuanthang** [comment]
Resolved in v0.3.1 🚀

https://github.com/tranxuanthang/lrcget/releases/tag/0.3.1

**Comment by dantob** [comment]
Would this make more sense to use the `#` comment tag instead of the `au` author tag?
